lib/naver/searchad/api/version.rb

```diff
@@ -1,7 +1,7 @@
 module Naver
   module Searchad
     module Api
-      VERSION = '1.0.0'
+      VERSION = '1.0.1'
```

Comment:
this changes quite crucial thing. I'd make it a minor or maybe even a major version bump.

Reply:
Yes, changed it to **1.1.0** now. :)